CHANGELOG.md

```diff
@@ -7,6 +7,16 @@ We follow the [Semantic Versioning 2.0.0](http://semver.org/) format.
 ## Unreleased
 
 
+## 1.4.0 - 2015-11-13
```

Comment:
NVM, @Scotchester  pointed out this is the generator and projects won't be pulling these automatically.